plugin.program.autowidget/resources/lib/common/utils.py

```diff
@@ -459,6 +464,38 @@ def cache_files(path, widget_id):
     _, _, changed = cache_expiry(hash, widget_id, add=files)
     return (files,changed)
 
+ERROR_FILE = {
+    "jsonrpc": "2.0", 
+    "id": 1, 
+    "result": {
+        "files": [
+            {
+                "title": "Error",
+                "label": "Error",
+                "file": "plugin://plugin.program.autowidget/?mode=force&refresh=&reload=",
+                "art": get_art('alert'),
+                "filetype": "file", 
+            }
+        ]
+    }
+}
+
+UPDATING_FILE = {
+    "jsonrpc": "2.0", 
+    "id": 1, 
+    "result": {
+        "files": [
+            {
+                "title": "Loading Content...",
+                "label": "Loading Content...",
+                "file": "plugin://plugin.program.autowidget/?mode=force&refresh=&reload=",
+                "art": get_art('alert'),
+                "filetype": "file", 
+            }
+        ]
+    }
+}
+
```

Comment:
Let's also move these structures to the top of the file, for organization.

Reply:
They were before but get_art is not defined yet so can't be.